docs/content/LabelGroup.md

```diff
@@ -0,0 +1,23 @@
+---
+title: LabelGroup
+---
+
+The LabelGroup component is used to add commonly used margins and wrapping for groups of Labels.
+
+## Default example
+
+```jsx live
+<LabelGroup>
+  <Label m={1}>Default label</Label>
+  <Label m={1} scheme="gray-darker">Darker gray label</Label>
+  <Label m={1} outline>Default outline label</Label>
+</LabelGroup>
```

Comment:
Can we get rid of the `m={1}` in the example?

Reply:
Oh yeah 😂